Mobile version completed

shakespear95 opened the pull request:
## Summary
- added the rest of the mobile version
- includes the following sections
    - Portfolio
    - Languages
    - Frameworks

OyePriscilla commented on index.html:
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
             </div>
         </div>
     </section> 
+    <br>
+    <br>
+    <br>
+    <section class="tonic">
+        <div class="tonicimage">
+            <img src="img/Snapshoot Portfolio.png" alt="snapshot">
+        </div>
+        <div class="tonichead">
+            <p>Tonic</p>
+        </div>
+        <ul class="toniclists1">
+            <li class="canopy">
```
- [x] Kindly take note that you are to wrap all the cards section in `main` tag and display it as grid. 😄 
![crr5](https://user-images.githubusercontent.com/103386183/198280225-67893911-e05f-4f73-a8c6-0762de7aafb1.png)

shakespear95 replied:
## resolved 
- I did not follow instructions to fix everything in the grid 
- I had to remove the <br> and use the grid for everything
- Issue resolved and it's even looking better 
- Thanks again

OyePriscilla commented on index.html:
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
+            <img src="img/Snapshoot Portfolio.png" alt="snapshot">
+        </div>
+        <div class="tonichead">
+            <p>Tonic</p>
```
- [x] Kindly ensure that all your cards' titles use header tag `hx` not `p` tag. 👍

OyePriscilla commented on index.html:
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
+           <br>
+            <div class="connect">
+                <div class="text3">
+                    <p>LET'S CONNECT </p>
```
- [ ] Kindly use header tag as well for this title. 😄

Kossi-stack replied:
- [x] Kindly change the p tag by a header tag

OyePriscilla commented on index.html:
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
+                    <div class="csstext">
+                        <p>CSS</p>
+                    </div>
+                </div>
```
- [x] Kindly ensure that the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li. 😄 
- [x] It is better to align them according to the template. 👍 

## Your Design 

![crr6](https://user-images.githubusercontent.com/103386183/198283919-ca6172e1-47d2-4264-86c2-cd88fda56083.png)


## Figma Design Template

![crr7](https://user-images.githubusercontent.com/103386183/198283995-443afa84-0151-4a1c-aa23-5465f9318afd.png)

shakespear95 replied:
Thank you for noticing this part, I had put too much space between the images and the text.
to fix it I had to reduce the margin that I had put in between the images and the text. 
thank you again.

Kossi-stack replied:
- [x] Kindly wrap all technology in a ul list and each of them in li

OyePriscilla commented on index.html:
```diff
@@ -69,5 +69,321 @@ <h2>I'm Shakespear Glad to see you!</h2>
+                        <p>CSS</p>
+                    </div>
+                </div>
+                <div class="frames">
+                    <p>Frameworks</p>
+                </div>
+                
+
+            <img src="img/Separator Bottom.png" alt="separater">
+
+            <div class="skillz">
+                <div class="skillstext">
+                    <p>Skills</p>
+                </div>
+                <div class="chevron">
+                    <img src="img/chevron-right.svg" alt="vector">
+                </div>
+            </div>
```
- [x] Please take note that the technology categories ("Languages", "Frameworks" and "Skills") are also in a `ul`, each of them in a `li`. 😄 
- [x] I noticed that the Framework category does not have chevron-right icon. Kindly add it. 👍

shakespear95 replied:
Thank you for noticing this error, I fixed it by adding another chevron-right icon. Issue fixed

OyePriscilla commented on css/style.css:
```diff
@@ -96,3 +96,399 @@ nav {
+  color: #172b4d;
+}
+
+.cardbut {
+  box-sizing: border-box;
+  flex-direction: column;
```
- [x] Kindly implement the buttons styling for hover, active and pressed. 👍 

## Figma Design Template
 
![crr8](https://user-images.githubusercontent.com/103386183/198285216-c09f63dd-e974-430b-b64e-f2e958629b8c.png)

- [x] Also, all your buttons are not properly aligned with other content; you may have to adjust the margin-left. 👍 


![crr9](https://user-images.githubusercontent.com/103386183/198286252-01bc99c3-9bc5-4a03-a377-dc69796b25d5.png)

shakespear95 replied:
## Resolved 👍 
- added the effects
- effects added
    - on hover
    - on press
    - active
- thanks for noticing

![Jack Sparrow Salute](https://media.giphy.com/media/2PmMiUc9OH4lO/giphy.gif)

OyePriscilla reviewed (changes requested):
## Hi @shakespear95 
Nice job, you are almost there.

![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
 
## (Highlights) Good Points: 
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

## (Changes Required) 

* [x]  Please check my inline comments for other changes required.
 
## [Optional] suggestions:

- [x] [optional] Please take note that commit messages should be in sentence case and be well descriptive. It would be great if you start implementing this with subsequent PRs.

> Having a good commit message history will allow you (or other developers working on your code) to quickly see what changes were made and why. This is useful if a bug is found in the code that needs to be fixed!


 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
**Cheers and Happy coding! :clap::clap::clap:**
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @oyepriscilla in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

shakespear95 commented:
summary of the corrections

- I corrected paragraph tag to headings  in the HTML files 
- I corrected the alignment of the pictures and text on the part of the languages
- added the chevron that I had missed
- used the grid instead of <be>
- changed my buttons to respond to hover, press, and active
- thanks for the review

Kossi-stack reviewed (changes requested):
## Hi @shakespear95,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966](https://user-images.githubusercontent.com/81444338/170121142-a9f93bb7-38bb-45cc-8327-aab2501ef90a.gif)

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions
- [OPTIONAL] Kindly use a proper naming for your branches.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

shakespear95 commented:
-errors fixed 

1. changing the paragraph tag to h2 
2. removing divs on lists and using un ordered lists

victorokoroji commented on index.html:
```diff
@@ -69,5 +69,310 @@ <h2>I'm Shakespear Glad to see you!</h2>
+            </div>
+           <br>
+           <br>
+            <div class="connect">
+                <div class="text3">
+                    <h1>LET'S CONNECT </h1>
+                </div>
```
- [x] As the previous reviewer suggested, kindly wrap your `LET'S CONNECT` text in an anchor tag as the previous reviewer has suggested 👍🏽 as the link should take you to the contact page or another page 👍🏽 

e.g.
```
<div class="text3">
         <a href="#">LET'S CONNECT </a>
   </div>
```

vickymarz reviewed (changes requested):
Hi @shakespear95, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## Top highlight 🎯

- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

vickymarz reviewed (approved):
Hi @shakespear95, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 
<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
